### DISCUSSION TITLE
How to fix @terminal @workspace command failure in chat

### DISCUSSION THREAD

**Discussion by tnt1223** [community/community#82166] [Copilot Conversations] [answered]
### Select Topic Area

Question

### Body

![image](https://github.com/community/community/assets/104182198/c8cc4e1c-c6dd-4a35-a5a7-cf19043f46d2)
when I ask something need to connect @terminal or @workspace, it raised an error. But It runs successfully for me to ask other questions, ruling out network reasons.

here are more details
![image](https://github.com/community/community/assets/104182198/b4f552dc-f6fe-4989-98e8-0d4c786e7f84)

**Comment by faridnec** [comment]
Hey have you tried close and reopen Visual Studio Code? Sometimes, a simple restart can resolve extension-related problems. Also maybe it is good to check that you are using the latest version of the Copilot extension. Go to the Extensions view `(Ctrl+Shift+X)`, search for "GitHub Copilot," and check if there are updates available.

if still persists u can try reinstall the extension. I hope this might help

**Reply by tnt1223** [reply]
yeah, I tried all of your advice before, and the verson of chat is v0.11.1, also latest.

**Reply by faridnec** [reply]
I'm so sorry for that,  could you provide additional information or context? If it really does not work -additional error message from the output terminal or problem - , you might want to consider opening a new issue on the repository or contact GitHub, describing the problem and including any relevant logs or error messages.

**Reply by tnt1223** [reply]
I don't know how to get further information, if you mean github copilot chat under the OUTPUT tab, the 2nd screenshot I uploaded was about that. Do I need to open the files one by one?

**Reply by tnt1223** [reply, marked as answer]
https://github.com/microsoft/vscode-copilot-release/issues/618
I tried to search the issue in English, and I found solution is starting a new session.
